### ISSUE TITLE
对east loss实现的疑惑

### ISSUE THREAD

**Issue by locusbear** [PaddlePaddle/PaddleOCR#8000]
看起来east的损失和论文中提到的不一样。论文中提到了回归损失rbox与quad，但代码实现中似乎只实现了quad，且quad的loss与论文公式似乎不一样。论文中是只计算了到四个顶点的smooth l1 loss，而代码实现是计算四个顶点的距离，并判定当在目标区域内其差值是否小于1，小于计算一个loss，大于计算一个loss。然后相加？为什么会有这么大的差异呢，另外还想了解(abs_geo_diff - 0.5) * (1.0 - smooth_l1_sign)和out_loss = l_geo_split[-1] / channels * in_loss * l_score的具体含义
